Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/ui/homecards/SubmissionState.kt

```diff
@@ -10,4 +15,9 @@ object TestNegative : SubmissionState()
 object TestError : SubmissionState()
 object TestInvalid : SubmissionState()
 object TestPending : SubmissionState()
-object SubmissionDone : SubmissionState()
+data class SubmissionDone(val testRegisteredOn: Date?) : SubmissionState() {
+
+    fun formatTestRegistrationText(context: Context): String =
```

Comment:
I wouldn't put formatting methods into data objects, but as I saw this also in other parts of the project I guess it's not a big deal.